Mathlib/Combinatorics/Quiver/Subquiver.lean

```diff
@@ -25,7 +25,7 @@ def WideSubquiver (V) [Quiver.{v + 1} V] :=
 
 /-- A type synonym for `V`, when thought of as a quiver having only the arrows from
 some `WideSubquiver`. -/
--- Porting note: no hasNonemptyInstance linter yet
+-- Porting note: no hasNonemptyInstance linter yet https://github.com/leanprover-community/mathlib4/issues/5171
```

Comment:
For the record: the formatting in 20824 is slightly different, and you're removing these notes in that PR.

That said: I think this change is an improvement in any case.